docs/modules/ROOT/pages/cops_style.adoc

```diff
@@ -9897,6 +9897,35 @@ end
 
 * https://rubystyle.guide#no-extend-struct-new
 
+== Style/SwapValues
+
+|===
+| Enabled by default | Safe | Supports autocorrection | VersionAdded | VersionChanged
+
+| Pending
+| Yes
+| Yes (Unsafe)
+| 1.1
+| -
+|===
+
+This cop enforces the use of shorthand-style swapping of 2 variables.
+It is marked as unsafe, because while autocorrecting it can erroneously remove
```

Comment:
The wording here is a bit misleading, as the cop is safe and only the auto-correct is unsafe.